### PR TITLE
Add social media links to footer.

### PR DESCRIPTION
**Pull request by dwilsonactual**
# 🔗 Issue
#640 

# ✍️ Description
We add icon links to the footer for social media URLs
With @ipc103

# 📷 Screenshots/Demos
image

### CODE REVIEW

File: app/views/layouts/shared/_footer.html.erb [hunk 1]
```diff
@@ -32,6 +26,49 @@
             <li>
               <%= active_link_to 'Partners', partners_path, class: 'nav-link' %>
             </li>
+            <% if Current.tenant&.profile.facebook_url.present? || Current.tenant&.profile.instagram_url.present? %>
+              <li class="d-flex gap-3">
+                <% if Current.tenant&.profile.facebook_url.present? %>
+                  <%= link_to Current.tenant&.profile.facebook_url, class: 'nav-link', target: '_blank' do %>
```
Comment (ipc103) [inline app/views/layouts/shared/_footer.html.erb:32]:
```suggestion
                  <%= link_to Current.tenant&.profile.facebook_url, class: 'nav-link', target: '_blank', aria: {label: "Find us on facebook" } do %>
```

**Comment by kasugaijin** [comment]
@all-contributors
please add @dwilsonactual for code.

**Comment by kasugaijin** [comment]
@all-contributors
please add @ipc103 for review.

**Review by kasugaijin — APPROVED** [review]
Looks good. Thank you @dwilsonactual

**Pull request merged**